_episodes/02-installation.md

```diff
@@ -25,6 +25,31 @@ be found in the
 We will first install Conda, and then ESMValTool. We end this chapter by testing
 that the installation was successful.
 
+
+> ## ESMValTool Installation/Usage  Options - A quick summary
+> Before we begin, here are all the possible ways in which you can use ESMValTool
+> depending on your level of expertise or involvement with ESMvalTool and 
+> associated software such as GitHub and Conda.
+>
+> 1) If you have access to a server where ESMValTool is already installed 
+> as a module, for e.g., the ``CEDA JASMIN`` server, you can simply
+> load the module and start using ESMValTool with the following :
+> ```bash
+> module load esmvaltool
+> ```
```

Comment:
```suggestion
> ```

After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
```